.semgrep/rpc_endpoint.yml

```diff
@@ -100,4 +107,4 @@ rules:
     severity: "WARNING"
     paths:
       include:
-        - "*_endpoint.go"
+        - "nomad/*_endpoint.go"
```

Comment:
This will exclude client endpoints, do they follow a different pattern for authorization now?

Reply:
There's two kinds of "client endpoints": the server-side client RPCs are in files like `nomad/client_*_endpoint.go` and get picked up with this glob. The client-side client RPCs in files like `client/*_endpoint.go` have always been excluded, because they don't match the first pattern with the RPC forwarding:

```
    patterns:
      - pattern: |
          if done, err := $A.$B.forward($METHOD, ...); done {
            return err
          }
```

This change just makes sure we don't bother checking the files that will never have this pattern.